packages/devtools/lib/src/service_manager.dart

```diff
@@ -226,6 +226,34 @@ class ServiceConnectionManager {
       isolateId: _isolateManager.selectedIsolate.id,
     );
   }
+
+  Future<int> getDisplayRefreshRate() async {
+    final flutterViewListResponse = await service.callServiceExtension(
+      registrations.flutterListViews,
+      isolateId: _isolateManager.selectedIsolate.id,
+    );
+    final List<dynamic> views =
+        flutterViewListResponse.json['views'].cast<Map<String, dynamic>>();
+
+    // Each isolate should only have one FlutterView.
+    final flutterView = views.firstWhere(
+      (view) => view['type'] == 'FlutterView',
+      orElse: () => null,
+    );
+
+    if (flutterView == null) {
+      print('No Flutter Views to query: ${flutterViewListResponse.json}');
+    }
+
+    final viewId = flutterView['id'];
```

Comment:
From above, `flutterView` might be `null` here.

Reply:
we should not reach this point now, see above.